src/modules/rover_ackermann/RoverAckermannGuidance/RoverAckermannGuidance.cpp

```diff
@@ -287,14 +287,12 @@ float RoverAckermannGuidance::updateAcceptanceRadius(const Vector2f &curr_wp_ned
 }
 
 float RoverAckermannGuidance::calcDesiredSteering(const Vector2f &curr_wp_ned, const Vector2f &prev_wp_ned,
-		const Vector2f &curr_pos_ned, const float &lookahead_gain, const float &lookahead_min, const float &lookahead_max,
-		const float &wheel_base, const float &desired_speed, const float &vehicle_yaw)
+		const Vector2f &curr_pos_ned, const float &wheel_base, const float &desired_speed, const float &vehicle_yaw)
```

Comment:
```suggestion
		const Vector2f &curr_pos_ned, const float wheel_base, const float desired_speed, const float vehicle_yaw)
```
And don't put the `const` in the function declaration (hpp) as it's not useful for the user of the library.

Reply:
changed in acd8e20